theseus/extlib/sparse_matrix_utils.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+from scipy.sparse import lil_matrix
+
+
+def generate_sparse_matrix(rows, cols, fill, min_entries_per_col):
```

Comment:
+1, I got confused too, until I saw this was doing fill in with 1.0.

Also, should we move this stuff to the primary utils dir since it doesn't depends on the cpp code?

Reply:
moving. well the generated object is a matrix, with just ones as values, maybe it's better `generate_mock_sparse_matrix`?

Reply:
In another comment I proposed `generate_random_binary_matrix` which seems more descriptive and more appropriate for a generic sparse matrix utility function, intended to be used beyond unit tests (given the python module it's located in).